docs/import-roster-from-lms.md

```diff
@@ -0,0 +1,78 @@
+## Importing a course roster from your Learning Management System
+
+GitHub Classroom is able to automatically import your course roster from your institution's Learning Management
+System, potentially saving you time and headaches when provisioning a new GitHub Classroom course!
+
+### Prerequisites
+
+This guide assumes that you've successfully [connected GitHub Classroom to you Learning Management System](/help/connect-to-lms). If you haven't, please follow those setup instructions before proceeding to the next steps.
+
+Importing a course roster from your Learning Management System will only work if you have you have enabled GitHub Classroom to receive course roster information per the privacy settings within your Learning Management System.
+
+### Creating a new course roster
+
+To import a new course roster from your Learning Management, select the classroom you would like to import into. Then proceed through the following steps:
+
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/roster-import/import-roster.gif" class="border" style="width: 75%;">
+   </div>
+
+1. Navigate to classroom settings within GitHub Classroom
+
+2. Select "Roster Management" from the sidebar
+
+3. On the "Add students" page which pops up, click "Import from your Learning Management System"
```

Comment:
A general tip for writing step-by-step instructions: I follow a mnemonic CAR—context, action, result—for writing individual steps. For example: 

> In the _Whatsit_ window, select the **Activate** button. A _Whosit_ appears.

Sometimes the context or action is omitted, if it's especially obvious or routine. For example, if a UI widget is large and prominent, you don't have to say "In the middle of the window". Likewise, you don't need to say what the result is for an obvious task, like entering some information in a text field or clicking a _Back_ button.

For this item in particular, the first bit ("The _Add students page_ loads.") is a result that goes with the previous step.